video/out/opengl/ra_gl.c

```diff
@@ -127,8 +127,9 @@ static int ra_init_gl(struct ra *ra, GL *gl)
     // Disable compute shaders for GLSL < 420. This work-around is needed since
     // some buggy OpenGL drivers expose compute shaders for lower GLSL versions,
     // despite the spec requiring 420+.
```

Comment:
Thinking out loud:

Does exposing the extension actually cause issues or is it to appease Nvidia's driver behaviour? If it's the latter a better approach is to match on "vendor == NVIDIA", so that all drivers are not set to the lowest common denominator set by Nvidia.

Reply:
In this case, I believe it's a problem with other vendors and not nvidia. Nvidia are very strict and hide the extension if the GL version is too low.

Reply:
In this case, I believe it's a problem with other vendors and not nvidia. Nvidia are very strict and hide the extension if the GL version is too low.

Reply:
2704625e3f9 leads #6272 which was reported on Windows/Intel and someone else says it's broken on Linux/Mesa too

Reply:
You're right - spec is pretty clear `OpenGL 4.2 is required.` Can I interest you in opening a [Mesa issue](https://gitlab.freedesktop.org/mesa/mesa/-/issues) so it gets fixed?